en/hardware/porting_guide.md

```diff
@@ -13,7 +13,7 @@ This guide is focused only on the host OS and middleware as the applications/fli
 In addition to the host operating system specific configuration files described below, there are several groups of configuration files for each board located throughout the code base:
```

Comment:
@dagar Can you please let me know correct paths for everything in this section (porting guide)